assets/scripts/db_filter_org.exs

```diff
@@ -0,0 +1,86 @@
+defmodule Glific.Migrations do
+  import Ecto.Query
+
+  alias Glific.{
+    Contacts.Contact,
+    Contacts.ContactsField,
+    Contacts.Location,
+    Extensions.Extension,
+    Flows.Flow,
+    Flows.FlowContext,
+    Flows.FlowCount,
+    Flows.FlowLabel,
+    Flows.FlowResult,
+    Flows.FlowRevision,
+    Flows.WebhookLog,
+    Groups.ContactGroup,
+    Groups.Group,
+    Groups.UserGroup,
+    Jobs.BigqueryJob,
+    Jobs.ChatbaseJob,
+    Jobs.GcsJob,
+    Messages.Message,
+    Messages.MessageMedia,
+    Partners.Credential,
+    Partners.Organization,
+    Repo,
+    Searches.SavedSearch,
+    Templates.SessionTemplate,
+    Tags.ContactTag,
+    Tags.MessageTag,
+    Tags.Tag,
+    Tags.TemplateTag,
+    Users.User
+  }
+
+  def execute do
+    [shortcode] = System.argv()
+
+    [organization_id] =
+      Organization
+      |> where([o], o.shortcode == ^shortcode)
+      |> select([o], o.id)
+      |> Repo.all(skip_organization_id: true)
+
+    [
+      BigqueryJob,
+      ChatbaseJob,
+      GcsJob,
+      Extension,
+      Credential,
+      ContactsField,
+      Location,
+      Tag,
+      TemplateTag,
+      WebhookLog,
+      FlowContext,
+      FlowCount,
+      FlowLabel,
+      FlowResult,
+      FlowRevision,
+      Flow,
+      MessageMedia,
+      MessageTag,
+      Message,
+      ContactGroup,
+      UserGroup,
+      Group,
+      ContactTag,
+      SavedSearch,
+      SessionTemplate,
+      User,
+      Contact
+    ]
+    |> Enum.each(fn module ->
+      module
+      |> where([q], q.organization_id != ^organization_id)
+      |> Repo.delete_all(skip_organization_id: true)
+    end)
+
+    Organization
+    |> where([q], q.organization_id != ^organization_id)
+    |> Repo.delete_all(skip_organization_id: true)
+  end
+end
+
```

Comment:
@anshulrr this can be a bit risky and can be run on production also. 

Let's use this function in the repo.

In the repo file and then 
```elixir

  if Mix.env() in [:dev, :test] do
    @spec truncate(Ecto.Schema.t()) :: :ok
    def truncate(schema) do
      table_name = schema.__schema__(:source)
      query("TRUNCATE #{table_name}", [])
      :ok
    end
  end

```

Glific.Repo.truncate(Organization)